### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by pointlessone**
* Updated PHP to 7.0
* Updated PHP Code Sniffer to 2.7.1
* Updated drupa/coder to 8.2.10

Because of changed base the image now is smaller. It was 44 MiB. Now it's 24.26 MiB.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,19 +1,34 @@
-FROM alpine:3.3
+FROM alpine:edge
```
Comment (GordonDiggs) [inline Dockerfile]: We've had trouble with `edge` before as it changes often. Would the `3.4` tag work?

Comment (pointlessone) [inline Dockerfile]: Weird enough, 3.4 doesn't have packages for PHP 7. So we either stuck with PHP 5 or edge Alpine.

**Comment by pointlessone** [comment]
@GordonDiggs This repo doesn't seem to run tests on PRs. Could you please take a look?

File: Dockerfile (hunk 1 above)
Comment (GordonDiggs) [inline Dockerfile]: Ah let's use edge then and then see what happens!

**Comment by GordonDiggs** [comment]
@pointlessone I updated the CircleCI config - can you try pushing again?

**Pull request merged**